packages/lit-dev-content/site/docs/components/shadow-dom.md

```diff
@@ -205,38 +205,47 @@ render() {
 
 For more information, see [HTMLSlotElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement) on MDN.
 
-### @queryAssignedNodes decorator { #query-assigned-nodes }
+### @queryAssignedElements and @queryAssignedNodes decorator { #query-assigned-nodes }
 
-The `@queryAssignedNodes` decorator converts a class property into a getter that returns all of the assigned nodes for a given slot in the component's shadow tree. The optional second boolean argument when true flattens the assigned nodes, meaning any assigned nodes that are slot elements are replaced with their assigned nodes. The optional third argument is a css selector which filters the results to matching elements.
+`@queryAssignedElements` and `@queryAssignedNodes` convert a class property into a getter that returns the result of calling
+[`HTMLSlot.assignedElements`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement/assignedElements) or [`HTMLSlot.assignedNodes`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement/assignedNodes) respectively on a given slot in the component's shadow tree.
+Use these to query the elements or nodes assigned to the given slot.
+
+Both accept an options object similar to what the native browser API accepts, with the addition of `slot`.
+Use `slot` to specify the slot to query, or leave undefined for the default slot.
```

Comment:
The first sentence forces people to go to the MDN docs, to discover that there's only one property in the native options object. Then the other two possible options are defined in text. As an alternative, I'd suggest a table or definition list listing all three:

`flatten`. Boolean specifying whether to flatten the assigned nodes by replacing any child `<slot>` elements with their assigned nodes.

`slot`. Slot name specifying the slot to query. Leave undefined to select the default slot.

`selector` (`queryAssignedElements` only). If specified, only return assigned elements that match this CSS selector.

Reply:
Great suggestion! Agree with the table approach and your structure. Addressing